apps/dht/src/dht.c

```diff
@@ -0,0 +1,78 @@
+/* dht.c - KNoT Application Client */
+
+/*
+ * Copyright (c) 2019, CESAR. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * The knot client application is acting as a client that is run in Zephyr OS,
+ * The client sends sensor data encapsulated using KNoT protocol.
```

Comment:
Explain what the example does. As you did in the confirmation message.

Reply:
Done